### PR TITLE
Add text span field

### PR DESCRIPTION
**Pull request by eduardbabayan**

### CODE REVIEW

File: pycognaize/document/field/span_field.py [hunk 1]
```diff
@@ -32,38 +33,60 @@ def __init__(self,
         tags = [] if tags is None else tags
         super().__init__(name=name, tags=tags,
                          group_key=group_key, confidence=confidence,
-                         group_name=group_name,)
+                         group_name=group_name, value=value)
+
         self._field_id = field_id
+        self._value = ''
+        # self._value = ' '.join([i.raw_value
```
Comment (hovhannes-zohrabyan) [inline pycognaize/document/field/span_field.py]: Remove commented line

File: pycognaize/document/field/span_field.py [hunk 2]
```diff
@@ -32,38 +33,60 @@ def __init__(self,
...
+    @property
+    def line_values(self):
+        return self._line_values
 
     @classmethod
     def construct_from_raw(cls, raw: dict, pages: Dict[int, Page],
                            html: Optional[HTML] = None,
-                           labels: ClassificationLabels = None)\
-            -> 'SpanField':
+                           labels: ClassificationLabels = None) -> 'SpanField':
```
Comment (hovhannes-zohrabyan) [inline pycognaize/document/field/span_field.py:52]: Please remove classification Labels form contructor

File: pycognaize/tests/test_document/test_field/test_span_field.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    def test__field_id(self):
+        self.assertEqual(self.span_field_1._field_id, self.raw_field_1['_id'])
+        self.assertEqual(self.span_field_2._field_id, self.raw_field_2['_id'])
+
+    def test_value(self):
+        self.assertIsInstance(self.span_field_1.value, str)
+        self.assertEqual(self.span_field_1.value, self.raw_field_1['value'])
+        self.assertEqual(self.span_field_2.value, self.raw_field_2['value'])
+
+    def test__construct_from_raw(self):
```
Comment (hovhannes-zohrabyan) [inline pycognaize/tests/test_document/test_field/test_span_field.py:64]: Please add tests for edge cases

File: pycognaize/common/enums.py [hunk 4]
```diff
@@ -73,6 +73,7 @@ class IqDocumentKeysEnum(enum.Enum):
     page = 'page'
     src = 'src'
     page_src = 'src'
+    text_span = 'text-span'
```
Comment (hovhannes-zohrabyan) [inline pycognaize/common/enums.py:76]: Please add changelog in CHANGELOG.md file and change version in __init__.py file and create badges

**Review by hovhannes-zohrabyan — APPROVED** [review]
